### PR TITLE
remove libc dependency

### PR DESCRIPTION
**Pull request by sendilkumarn**
use `std::os::raw` instead

`wasm` compilation is happy with this.

r? @Yoric

### CODE REVIEW

File: src/opus/imported_decode.rs [hunk 1]
```diff
@@ -24,12 +23,12 @@ pub type ec_window = opus_uint32;
 pub struct ec_dec<R> where R: std::io::Read {
     pub inp: R,
     pub end_window: ec_window,
-    pub nend_bits: libc::c_int,
-    pub nbits_total: libc::c_int,
+    pub nend_bits: std::os::raw::c_int,
```
Comment (Yoric) [inline src/opus/imported_decode.rs:26]: Could you take the opportunity to replace all these `std::os::raw::*` with the pub type defined above?

**Review by Yoric — APPROVED** [review]
Looks good to me!

I think that it won't change anything. Unfortunately, we don't have tests that would catch regression, but I can't think of any risk.

File: src/opus/imported_decode.rs (hunk 1 above)
Comment (sendilkumarn) [reply src/opus/imported_decode.rs:26]: I don't get it

Comment (Yoric) [reply src/opus/imported_decode.rs:26]: Nevermind, I'll do it in a followup.

**Comment by Yoric** [comment]
Tell me when you're ready to land!

**Comment by sendilkumarn** [comment]
If you are changing in the follow up. I think it is ready to merge

**Pull request merged**